Twig/Extension/SonataAdminExtension.php

```diff
@@ -437,6 +437,13 @@ protected function getTemplate(
         try {
             $template = $environment->loadTemplate($templateName);
         } catch (\Twig_Error_Loader $e) {
+            @trigger_error(
+                'Relying on default template loading on field template '.
+                'loading exception is deprecated since 3.x and will be '.
+                'removed in 4.0, and a \Twig_Error_Loader exception will be '.
```

Comment:
There are two limits, none of which are hard. Let me use 80, which is better, and I'll let you use 120.